### ISSUE TITLE
Analytics: Implement analytics 

### ISSUE THREAD

**Issue by kevinp** [DrivingThru/DrivingThruiOS#79]
Follows #78 

Push same events defined in #74 into analytics as goals/events

@vfxdrummer , since we are interested in defining "domain events" and taking x number of actions in response, do we need basic pub/sub? #76

**Comment by kevinp** [comment]
We will implement Google Analytics for Firebase
https://firebase.google.com/docs/analytics/

**Comment by kevinp** [comment]
We will implement Google Analytics for Firebase
https://firebase.google.com/docs/analytics/

**Comment by vfxdrummer** [comment]
Yeah, I built an AnalyticsManager. It can serve as a PubSub model, if need be.

**Comment by siegel86** [comment]
@vfxdrummer is this still open?
